echemdb/data/cv/entry.py

```diff
@@ -207,12 +208,20 @@ def y(self):
         """
         from astropy import units as u
 
-        if u.Unit(self.figure_description.current.unit).is_equivalent('A / m2'):
-            return 'j' 
-        if u.Unit(self.figure_description.current.unit).is_equivalent('A'):
-            return 'I'
-        else:
-            raise ValueError(f"The variable on the y-axis is not equivalent to 'A / m2' or 'A'.")
+        if not self.data_description.axes:
+            raise IndexError("No axes were specified for the dataset.")
```

Comment:
```suggestion
            raise Exception("No axes were specified for the dataset.")
```

An index error should be raised when something (e.g. a list) is accessed out-of-bounds. If in doubt, just use `Exception`.